### ISSUE TITLE
Accessibility: ListPeoplePicker never reads out when the data is completed loading

### ISSUE THREAD

**Issue by rickyp-ms** [OfficeDev/office-ui-fabric-react#2345]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: tasks.office.com

#### Describe the issue:

Accessibility review opened a bug on us...
"Voice Over doesn't announce anything for the auto suggestions and if no results are found during search."

#### Actual behavior:

Reviewing this in Edge, it does say "Status loading" but it never actually says "Status loaded - # results" which leave a blind user thinking it's loading forever.

#### Expected behavior:

Screen reader to read out when the action is done so the user can continue with their scenario

**Comment by rickyp-ms** [comment]
Looks like in the latest code seems it says "suggestions available" properly. Using this to add code for the no results found scenario.
